docs/sessions/dapr_session.md

```diff
@@ -0,0 +1,670 @@
+# Dapr State Store Sessions
```

Comment:
Can you remove this document page? We'd like to keep the number of sub pages here minimal. Instead, you can add more guides in the example code comments if necessary.